finite_state_sdk/queries.py

```diff
@@ -471,7 +550,7 @@ def _create_GET_FINDINGS_VARIABLES(asset_version_id=None, category=None, cve_id=
         __typename
     }
 }""",
-    "variables": lambda asset_version_id=None, category=None, cve_id=None: _create_GET_FINDINGS_VARIABLES(asset_version_id=asset_version_id, category=category, cve_id=cve_id)
+    "variables": lambda asset_version_id=None, category=None, cve_id=None, status=None, severity=None, limit=None: _create_GET_FINDINGS_VARIABLES(asset_version_id=asset_version_id, category=category, cve_id=cve_id, severity=severity, status=status, limit=limit)
```

Comment:
Similar to the previous comment, consider separating the lambda expression into its own function for improved readability and maintainability.